site/content/blogs/release-v127.en.md

```diff
@@ -67,7 +182,7 @@ to your designated directory.
 
 ##### `copilot [noun] deploy --diff`
 
-Similar to `copilot [noun] package --diff`, you can run `copilot [noun] deploy --diff` to see the same diff. 
+Similar to `copilot [noun] package --diff`, you can run `copilot [noun] deploy --diff` to see the same diff.
 However, instead of exiting after it print the diff, Copilot will follow up with a question: `Continue with the deployment? [y/N]`.
```

Comment:
```suggestion
However, instead of exiting after it prints the diff, Copilot will follow up with the question: `Continue with the deployment? [y/N]`.
```